### PR TITLE
Added string argument support for Model::validate() #11368

### PR DESCRIPTION
**Pull request by lynicidn**
| Q | A |
| --- | --- |

Is bugfix?  parially
New feature?    no
Breaks BC?  no
Tests pass?     yes
Fixed issues    #11368

### CODE REVIEW

**Comment by SilverFire** [comment]
| Q             | A
| ------------- | ---
| Is bugfix?    | parially
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #11368

File: framework/helpers/BaseHtml.php [hunk 1]
```diff
@@ -2065,7 +2065,7 @@ public static function getAttributeValue($model, $attribute)
             throw new InvalidParamException('Attribute name must contain word characters only.');
         }
         $attribute = $matches[2];
-        $value = $model->$attribute;
+        $value = ArrayHelper::getValue($model, $attribute);
```
Comment (SilverFire) [inline framework/helpers/BaseHtml.php]: Why?

**Comment by SilverFire** [comment]
On hold until discussion end in #11368

**Comment by lynicidn** [comment]
it's merge :( after git fetch upstream

**Comment by lynicidn** [comment]
>Breaks BC? 	no

i think it break BC

**Comment by SilverFire** [comment]
> i think it break BC

Behavior for `string` is neither declared nor tested and brings unexpected result. I think it's not a BC

**Comment by SilverFire** [comment]
Sorry, this PR will not be merged

**Pull request closed without merging**